nest-tax-backend/src/tasks/__tests__/tasks.service.spec.ts

```diff
@@ -14,6 +14,7 @@ describe('TasksService', () => {
   let service: TasksService
 
   beforeEach(async () => {
+    process.env.FEATURE_TOGGLE_REMINDER_UNPAID_TAX = 'false'
```

Comment:
This should have been `true`, as we still want to test the functionality in case, this would be true in real life. We don't need to test the functionality if it is false, because it is a very temporary workaround, so this would be sufficient. No other changes in this files needed.